src/dnddiceroller/DiceTower.java

```diff
@@ -15,8 +15,9 @@
  * @author Paul Scarrone
  */
 public class DiceTower {
-  final int PANEL_COUNT = 3;
-  List<Die> dice;
+  final private int PANEL_COUNT = 3;
```

Comment:
I'll go in and change this. Ill update the commit once I can figure out why my roll method won't work.